tool/tctl/common/user_command.go

```diff
@@ -245,7 +245,7 @@ func printTokenAsText(token services.ResetPasswordToken, messageFormat string) e
 		return trace.Wrap(err, "failed to parse reset password token url")
 	}
 
-	ttl := token.Expiry().Sub(time.Now().UTC()).Round(time.Second)
+	ttl := trimDurationZeroSuffix(token.Expiry().Sub(time.Now().UTC()).Round(time.Second))
```

Comment:
move the rounding into `trimDurationZeroSuffix`, to make it play nice with durations containing shorter units